src/features/CourtCaseDetails/Tabs/Panels/Offences/Offence/OffenceDetails.tsx

```diff
@@ -226,6 +227,16 @@ export const OffenceDetails = ({
           )}
           <TableRow label="Court offence sequence number" value={offence.CourtOffenceSequenceNumber} />
           <TableRow label="Committed on bail" value={getCommittedOnBail(offence.CommittedOnBail)} />
+          <ConditionalRender
+            isRendered={
+              offence.Result.length > 0 &&
+              offence.Result[0].PleaStatus !== undefined &&
+              offence.Result[0].Verdict !== undefined
+            }
+          >
+            <TableRow label="Plea" value={getPleaStatus(offence.Result[0].PleaStatus)} />
+            <TableRow label="Verdict" value={getVerdict(offence.Result[0].Verdict)} />
+          </ConditionalRender>
```

Comment:
Wondering if Plea/Verdict should be rendered independently in case one is missing and the other is present.

Reply:
I think they probably should if that's what the zod schema says